VikingQuiz.Api/Repositories/GameRepo.cs

```diff
@@ -53,6 +54,27 @@ public Game GetGameById(int id)
         {
             return ctx.Game.Find(id);
         }
+
+        public Game GetGameByCode(string code)
+        {
+            Game foundGame = ctx.Game.Where(x => x.Code == code)
+                .Select(x => new Game { Id = x.Id, QuizId = x.QuizId, GameDate = x.GameDate, Code = x.Code })
```

Comment:
A mapping method would have looked more nice here instead of creating the Game inside the select. Also, please use meaningful names for lambda function parameters